docs/sections/examples.md

```diff
@@ -126,7 +126,50 @@ taking advantage of each of their individual strengths.
 
 ### On CloudLab using geni-lib
 
-**TODO**
+[CloudLab](https://www.cloudlab.us/) provides researchers with control and
+visibility all the way down to the bare metal.
+
+Most **CloudLab** resources provide hard isolation from other users, so it can
+support hundreds of simultaneous ***slices***, with each getting an artifact-free
+environment suitable for scientific experimentation with new cloud architectures.
+Run standard cloud software stacks such as *OpenStack* and *Hadoop*. 
+Or, build your own from the ground up. The bare metal's the limit!
+
+**CloudLab** is built from the software technologies that make up [Emulab](emulab.net) and
+ parts of **GENI**, so it provides a familiar, consistent interface for researchers.
```

Comment:
We can have a brief summary of CloudLab and point to cloudlab.us and docs.cloudlab.us . Something like "CloudLab is a NSF-sponsored baremetal as a service provider the gives students and researchers access to cloud resources so they can setup clusters of machines to execute experiments in many domains such as networking, computer systems, cloud systems, etc."

And then in a new paragraph: "The following is an [example of a `CloudLab ready` pipeline](https://github.com/popperized/popper-readthedocs-examples/tree/master/pipelines/cloudlab-benchmarking), i.e. a pipeline that can be executed on CloudLab with minimal effort. To execute this pipeline:"